solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -192,32 +192,41 @@ export default {
     created() {
         let location = ""
         if (window.location.hash) {
-            let section = window.location.hash.substring(1).replace("-", ".");
-            if (section.includes("-")) {
-                // eslint-prefer-destructuring, kinda cool
-                [section] = section.split("-");
-            }
-            if (isNaN(section)) {
-                location = `locations=${this.title}.${this.part}.${section}`
-            }
-            else {
-                location = `locations=${this.title}.${section}`
-                this.selectedPart = `§ ${section}`;
-        }
-        } else {
+          location = this.parseHash(window.location.hash)
+         } else {
             this.fetch_content(this.title, this.part);
         }
         this.fetch_content(this.title, this.part, location)
+        window.addEventListener('hashchange', (event) => {
+          location = this.parseHash(window.location.hash)
+          this.fetch_content(this.title, this.part, location)
+        }, false);
     },
-
     mounted() {
         this.$root.$on(EventCodes.SetSection, (args) => {
             this.selectedPart = args.section;
         });
         this.categories = getDefaultCategories();
     },
+    destroyed() {
+      window.removeEventListener("hashchange", this.hashchange);
```

Comment:
`this.hashchange` is undefined because it's not a property defined anywhere in the component.  

The second argument here needs to be the exact same function instance that was passed into the `addEventListener` method above.  See description and examples here:

https://deepscan.io/docs/rules/bad-remove-event-listener